### DISCUSSION TITLE
Image Not Displaying on Organization Profile: Bug or Oversight?

### DISCUSSION THREAD

**Discussion by siMobin** [community/community#120527] [Repositories]
### Select Topic Area

Bug

### Body

I just set up my organization profile repository (.github). But the image is not showing on the profile. My question is, is this a bug or did I miss something?
Here is the screenshot.
### Main repo
https://github.com/CultureHubBD/.github
![image](https://github.com/community/community/assets/102217786/8b1862a2-d61b-46a8-bef3-245d65f123ac)

### Profile
https://github.com/CultureHubBD
![image](https://github.com/community/community/assets/102217786/5ef32393-fffd-469a-b8e2-f2f7a750af19)

**Comment by gurjeetsinghvirdee** [comment]
Hey @siMobin 
It's not working because in a `md` file, you can't upload an image like this `/assets/image.png`. Markdown doesn't work like other languages where you can directly add images from your **asset** folder. You need to use a  your asset location `GitHub URL` like this: `https://github.com/CultureHubBD/.github/blob/main/assets/Logo.ai`

**Reply by siMobin** [reply]
But it works on personal profile README file....! @gurjeetsinghvirdee 
check it out https://github.com/siMobin

**Reply by gurjeetsinghvirdee** [reply]
because it was in same directory in your readme profile all things store in a single place,
but in your organization case the assets are stored in different directory`(md doesn't use relative paths)`
